backend/src/main/java/com/zpi/backend/reservation_status/ReservationStatusRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.zpi.backend.reservation_status;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ReservationStatusRepository extends JpaRepository<ReservationStatus, Long> {
+    ReservationStatus findByStatus(String status);
```

Comment:
It should return Optional status (when I typed status 8, there was 500 error, null pointer exception)